pkg/api/types.go

```diff
@@ -1930,6 +1942,8 @@ type PodStatus struct {
 	// when we have done this.
 	// +optional
 	ContainerStatuses []ContainerStatus `json:"containerStatuses,omitempty"`
+	// +optional
+	Qos PodQOSClass `json:"qos,omitempty"`
```

Comment:
Even though @vishh wanted `qos`, it's too terse for an API that novice users might use.  I don't think adding `Class` will impact experienced users.